internal/service/searchdeployment/data_source_search_deployment_schema.go

```diff
@@ -0,0 +1,62 @@
+package searchdeployment
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:            true,
+				Description:         "Unique 24-hexadecimal digit string that identifies the search deployment.",
+				MarkdownDescription: "Unique 24-hexadecimal digit string that identifies the search deployment.",
+			},
+			"cluster_name": schema.StringAttribute{
+				Required:            true,
+				Description:         "Label that identifies the cluster to return the search nodes for.",
+				MarkdownDescription: "Label that identifies the cluster to return the search nodes for.",
```

Comment:
will scaffolding used only for new TF plugin or also for older one?  just to understand if it's enough to generate MarkdownDescription or we also want to generate Description

Reply:
The tool is limited to generating new TF framework code. I have left both definitions as they are both generated by the tool and provide unique value as per comments in the terraform code:

> - MarkdownDescription: used in various tooling, like the documentation generator
> - Description: used in various tooling, like the language server, to give practitioners more information about what this attribute is, what it's for, and how it should be used.